src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,89 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService service;
+
+    @BeforeEach
+    void init() {
+        Storage.people.clear();
+        service = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_valid_user_Ok() {
+        User expected = new User("validLogin", "123456789", 25);
+        User actual = service.register(expected);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void register_bulk_valid_users_Ok() {
+        User first = new User("validLogin", "123456789", 25);
+        User actual = service.register(first);
+        User expected = first;
+        assertEquals(expected, actual);
+
+        User second = new User("validLogin2", "123456789", 35);
+        actual = service.register(second);
+        expected = second;
+        assertEquals(expected, actual);
+
+        User third = new User("validLogin3", "123456789", 30);
+        actual = service.register(third);
+        expected = third;
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void register_null_user_notOk() {
```

Comment:
too much underscors. please read requirements for your method names one more time. Fix the same issue in all places